source/iterators.tex

```diff
@@ -4106,6 +4140,537 @@
 \tcode{insert_iterator<Container>(x, i)}.
 \end{itemdescr}
 
+\rSec2[const.iterators]{Constant iterators and sentinels}
```

Comment:
Should we follow the convention of [move.iterators] here? (i.e. add subclause like General, construction and assignment, etc.)

Reply:
I agree that the current presentation is a bit awkward. I'll try something. Thanks!